apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.environment-variables/route.tsx

```diff
@@ -214,15 +214,13 @@ export default function Page() {
           )}
           <Table containerClassName={cn(environmentVariables.length === 0 && "border-t-0")}>
             <TableHeader>
-              <TableRow>
-                <TableHeaderCell>Key</TableHeaderCell>
-                {environments.map((environment) => (
-                  <TableHeaderCell key={environment.id}>
-                    <EnvironmentLabel environment={environment} />
-                  </TableHeaderCell>
-                ))}
-                <TableHeaderCell hiddenLabel>Actions</TableHeaderCell>
-              </TableRow>
+              <TableHeaderCell>Key</TableHeaderCell>
+              {environments.map((environment) => (
+                <TableHeaderCell key={environment.id}>
+                  <EnvironmentLabel environment={environment} />
+                </TableHeaderCell>
+              ))}
+              <TableHeaderCell hiddenLabel>Actions</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"